system/handlers/formcontrols/FormBuilderFormPicker.cfc

```diff
@@ -0,0 +1,17 @@
+component {
+
+	public string function index( event, rc, prc, args={} ) {
+		arguments.args.labels = [ "" ];
+		arguments.args.values = [ "" ];
+
+		var forms = getPresideObject('formbuilder_form').selectData( filter={ active=1 } );
```

Comment:
Prefer `active=true`. 

I'll merge this anyway and just updated it. But future note, if the field is boolean, we should use a strict boolean value rather than relying on knowing specific DB implementation is actually a bit field.

Reply:
Noted. Thanks.